src/init/ext_manifest.c

```diff
@@ -32,6 +32,7 @@ const struct ext_man_fw_version ext_man_fw_ver
 #endif
 		.tag = SOF_TAG,
 		.abi_version = SOF_ABI_VERSION,
+		.fw_hash = SOF_FW_HASH,
```

Comment:
Is this reproducible - i.e are you hashing any generated files that could differ for different users.